src/lib/unregistered-services-healthCheck.js

```diff
@@ -0,0 +1,36 @@
+let lastCheckOk = true;
+let lastCheckOutput = 'All services\' metrics are set up in Next-metrics';
+let panicGuide = 'Don\'t panic';
+let lastCheckTime;
+
+module.exports = {
+	setAppName: appName => {
+		return {
+			getStatus: () => {
+				return {
+					name: `All services for ${appName} are registered in Next-metrics`,
+					ok: lastCheckOk,
+					checkOutput: lastCheckOutput,
+					lastUpdated: lastCheckTime,
+					panicGuide: panicGuide,
+					severity: 3,
```

Comment:
Is this correct severity?

Reply:
I can never remember if 3 is the most severe or least :joy:

If the existing error rate check is 3, then that means 3 is probably most severe. 

Your new check should be least severe, so if the error rate check is 3, make the new one a 1